### DISCUSSION TITLE
predictions_with_good_interpae.csv

### DISCUSSION THREAD

**Discussion by DimaMolod** [KosinskiLab/AlphaPulldown#325] [Ideas]
### Table layout

Currently the table has these headings
<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:////Users/dmolodenskiy/Library/Group%20Containers/UBF8T346G9.Office/TemporaryItems/msohtmlclip/clip.htm">
<link rel=File-List
href="file:////Users/dmolodenskiy/Library/Group%20Containers/UBF8T346G9.Office/TemporaryItems/msohtmlclip/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">

jobs | iptm_ptm | iptm | pDockQ/mpDockQ | average_interface_pae | average_interface_plddt | binding_energy | interface | Num_intf_residues | Polar | Hydrophobhic | Charged | contact_pairs | sc | hb | sb | int_solv_en | int_area | pi_score | pdb | pvalue
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --

</body>

</html>

While the last values derived from ccp4 programs are always empty:
<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:////Users/dmolodenskiy/Library/Group%20Containers/UBF8T346G9.Office/TemporaryItems/msohtmlclip/clip.htm">
<link rel=File-List
href="file:////Users/dmolodenskiy/Library/Group%20Containers/UBF8T346G9.Office/TemporaryItems/msohtmlclip/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">

interface | Num_intf_residues | Polar | Hydrophobhic | Charged | contact_pairs | sc | hb | sb | int_solv_en | int_area | pi_score | pdb | pvalue
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --

</body>

</html>

We must fix this and (partially) remove ccp4 values that are not informative (polar,hydrophobic,charged,etc?).
We may also consider removing pDockQ/mpDocQ.
### PyRosetta
The typical binding energies from pyRosetta are huge negative numbers expressed in Rosetta arbitrary units. These units correlate with Gibbs free energy, but to convert these numbers to more standard energy units like J/mol or kcal/mol, one must calibrate these numbers using, e.g., experimental data. 
As it might be too complicated and beyond the scope of AP, we can calibrate using some complexes with very well-known strong affinity (e.g. this https://www.rcsb.org/structure/1STP). Then, we can calibrate the rosetta units to the range [0,1].

**Comment by DimaMolod** [comment]
The main issue that the output csv table is broken is still remains

**Comment by jkosinski** [comment]
Some ideas from my side:

iptm_ptm - keep ofc
iptm	- keep ofc
pDockQ/mpDockQ	- keep - it nicely highlights models with good iptm but poor plddt at the interface. Consider checking whether it still applies to multimer v3 (was trained on v2 ?). Consult with authors? This might be a highly requested score because of its popularity
average_interface_pae - keep
average_interface_plddt - keep
binding_energy - to discuss. this is Rosetta? we should do at least a sanity check whether results are not random and whether high or low is good. No need to rescale to units, just treat it as a score. Rename to Rosetta_binding_energy. It might be that the score is not suitable for AlphaFold model if it explodes to high values on clashes.
interface	- what is that?
Num_intf_residues	- keep as should be approximately applicable to theoretical models?
Polar	- keep as should be approximately applicable to theoretical models?
Hydrophobhic	- keep - correct typo ;-)
Charged	- keep as should be approximately applicable to theoretical models?
contact_pairs	- keep as should be approximately applicable to theoretical models?
sc - keep as should be approximately applicable to theoretical models?
hb - remove as requires higher precision and changes a lot depending whether relaxation is done?
sb - remove as requires higher precision and changes a lot depending whether relaxation is done?
int_solv_en - keep as should be approximately applicable to theoretical models?
int_area - keep as should be approximately applicable to theoretical models?
pi_score - consider removing? We are not sure whether it works well for AF models in general and it struggles with small interface (as the authors admit)
pvalue - remove? do we use it? it is a pisa score that might need high precision structures?
